webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -65,12 +65,39 @@ export default class SidebarRight extends React.PureComponent {
         this.props.actions.setVisible(false);
     }
 
+    installationButtons(installation) {
+        return (
+            <div>
+                <Button
+                    className='btn btn-primary btn-sm'
+                    onClick={() => window.open('https://' + installation.DNSRecords[0].DomainName, '_blank')}
+                >{'Open'}
+                </Button>
+                <DropdownButton
+                    style={style.dropdownButton}
+                    className='btn btn-tertiary btn-sm'
+                    title='Logs'
+                >
+                    <MenuItem
+                        onClick={() => window.open(installation.InstallationLogsURL, '_blank')}
+                    >{'Installation Logs'}
+                    </MenuItem>
+                    <MenuItem
+                        onClick={() => window.open(installation.ProvisionerLogsURL, '_blank')}
+                    >{'Provisioner Logs'}
+                    </MenuItem>
```

Comment:
Small optimization if you want - declare something like 

```
const dropdownButtonItems [
   {onClick: () => window.open(installation.InstallationLogsURL, '_blank'), buttonText: 'Installation Logs'}
   ...
   ...
]
```

Then you can use `installationButtons.map`, and new buttons won't need to touch the JSX